scripts/integration-tests.sh

```diff
@@ -42,6 +47,8 @@ echo "Installing kubecfg"
 install_kubecfg
 echo "Installing Kubeless"
 install_kubeless
+echo "Installing Nats"
+install_nats
```

Comment:
thanks for the extra work of installing NATS in the test environment but it's actually not necessary for the unit test you added. We added Kafka for the test `should get a submmited message "hello world"` from `./test/examples-test.js` (that actually deploys a function) but it's better to not do the same with NATS since the Minikube environment in which the tests are being executed has very limited resources. Do you mind removing this installation? The unit test will work without this since it mocks the HTTP endpoint.

Reply:
@andresmgot Sure, has been done ^^